core/assets/sounds/cues/loveRap/data.json

```diff
@@ -0,0 +1,705 @@
+{
```

Comment:
Why is Love Rap here? I rejected the old pull request. Please rebase to the latest master commit.